lib/config/configuration.go

```diff
@@ -2747,6 +2747,14 @@ func Configure(clf *CommandLineFlags, cfg *servicecfg.Config, legacyAppFlags boo
 		cfg.Proxy.QUICProxyPeering = true
 	}
 
+	if rawPeriod := os.Getenv("TELEPORT_UNSTABLE_AGENT_ROLLOUT_SYNC_PERIOD"); rawPeriod != "" {
```

Comment:
We will need this when we'll do end to end testing in cloud and check if rollout happens. This will allow us to reduce the test duration from hours to minutes.

Reply:
removed